docs/introduction/visualization.rst

```diff
@@ -0,0 +1,50 @@
+Visualization
+=============
+
+The Objectypes API works together with the Objects API, objects of a certain
```

Comment:
It looks like the second part of this sentence is not finished?

Reply:
Slight change made.